network-api/networkapi/buyersguide/pagemodels/products/base.py

```diff
@@ -334,6 +349,15 @@ def specific(self):
         symmetrical=False
     )
 
+    def get_related_products(self):
+        """
+        This function is used by our custom RelatedProductFieldPanel, to make sure
+        we don't list every single PNI product ever entered into the system, but only
+        products added in recent iterations of PNI. For PNI v4 this has been set to
+        "any product entered after 2019".
+        """
+        return Product.objects.filter(review_date__gte=date(2020, 1, 1)).order_by('name')
```

Comment:
note that this has to be a `date`, not a `datetime`, as `review_date` is an integer (yay!)